### ISSUE TITLE
Next Button

### ISSUE THREAD

**Issue by ntantum** [CollegeOfTheHolyCross/dta_sufia#118]
We have recently been thinking that a "Next" and "Previous" button would be helpful to have on the DTA. For example, when a user enters a search term and clicks on an item, they would be able to click a "next" button to see the next item in their search, instead of having to click back out and find the next item in their original search list. This could also apply when looking at a collection. Once the user has clicked an item in a collection, it might be helpful to have a next button to view the next item in the collection, instead of going back to the collection home page.

Let me know if you think this function would be feasible on the website! Thanks.

**Comment by scande3** [comment]
@ntantum -> It should be possible and I'll do some research on the complexity tomorrow. Thanks!

**Comment by scande3** [comment]
Added as part of commit c6a52ab53f79339d64277e3127ca57b8cc967e54 . Let me know if it needs visual adjustment or if you notice any bugs with the functionality.

**Comment by ntantum** [comment]
This is great! I am very excited about having this button on the site! Thanks for doing that so fast.

Is there any way we could duplicate the button at the bottom of the page as well? Just for easier access.

**Comment by scande3** [comment]
Added to the bottom with commit e32451b81b6528c6dadd09cf7bbfc770b426e68f . Let me know if there are changes needed for it.

**Comment by kjrawson** [comment]
That's perfect. Thanks!!

**Comment by ntantum** [comment]
Unfortunately, our beloved next button has been having some issues. One of our students has been doing a lot of searching and using the next button but sometimes the items show up out of order. For example, she has been searching "latino" and after pressing next on result #33 and #34, the next button brings her to a collection page (https://www.digitaltransgenderarchive.net/collections/v692t6278) instead of object #35.

I additionally realized that this error has been popping up sometimes with the next button. I think this might be connected to the issue of the collection page from above. For example, when the next button brings us to a collection page, I have backed back out of the collection and tried to click on the item (in the above instance, #35) from the search page and it has brought me straight to this error message.

<img width="1082" alt="screen shot 2017-10-25 at 11 11 04 am" src="https://user-images.githubusercontent.com/30127831/32006710-804ee5a6-b975-11e7-8230-48d2e251c001.png">

I have personally had a few other issues with the next button presenting items out of order or just completely jumping to a new number (ex. result 10 of 100 will randomly change to result 23 out of 150). Unfortunately, I did not record these instances at the time but I will post more examples as I encounter them.

Hopefully this makes sense! Let me know if you have any questions!
